### PR TITLE
Release v0.16.3

### PR DESCRIPTION
**Pull request by ni4**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 ## Changelog
 
+### 0.16.3 [2023-04-11]
+
+#### Security
+
+* Fix for CVE-2023-29479 : possible hang on malformed input.
```
Comment (ribose-jeffreylau) [inline CHANGELOG.md:7]: The following might be more aligned with previous entries regarding fixed CVE:

```diff
-* Fix for CVE-2023-29479 : possible hang on malformed input.
-* Fix for CVE-2023-29480 : in some cases secret keys remain unlocked after use.
+* Fixed issue with possible hang on malformed inputs (CVE-2023-29479).
+* Fixed issue where in some cases, secret keys remain unlocked after use (CVE-2023-29480).
```

**Review by ribose-jeffreylau — CHANGES_REQUESTED** [review]

**Review by ronaldtse — APPROVED** [review]
Feel free to merge when @ribose-jeffreylau approves. Thanks!

**Comment by ni4** [comment]
> Feel free to merge when @ribose-jeffreylau approves. Thanks!

Should we care about the CI failures here? Those must be caused by updated GitHub environment/runners. Locating and picking related commits from the main to the release branch will be quite messy.

**Comment by ronaldtse** [comment]
Yes we need to fix the CI issues.

**Review by ribose-jeffreylau — APPROVED** [review]

**Comment by ronaldtse** [comment]
Thank you @ni4 ! Merging now.

**Pull request merged**